pkg/services/librarypanels/librarypanels_test.go

```diff
@@ -15,78 +16,251 @@ import (
 )
 
 func TestCreateLibraryPanel(t *testing.T) {
-	libraryPanelScenario(t, "When an admin tries to create a library panel", "then it should fail if library panel already exists", func(t *testing.T) {
-		lps, context := setupTestEnv(t, models.ROLE_ADMIN, map[string]string{})
-		command := getCreateCommand(1, "Text - Library Panel")
+	t.Run("When an admin tries to create a library panel that already exists", func(t *testing.T) {
+		t.Run("then it should fail", func(t *testing.T) {
```

Comment:
You should only use `t.Run` to define (sub) test cases, not to group assertions.
```suggestion
```

Reply:
Okidoki

Reply:
Fixed